modules/junit-jupiter/src/test/java/org/testcontainers/junit/jupiter/TestLifecycleAwareContainerMock.java

```diff
@@ -0,0 +1,43 @@
+package org.testcontainers.junit.jupiter;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.lifecycle.TestDescription;
+import org.testcontainers.lifecycle.TestLifecycleAware;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class TestLifecycleAwareContainerMock extends GenericContainer implements TestLifecycleAware {
```

Comment:
FYI the jupiter module works with any `Startable`, so, in theory, there is no need to extend it from `GenericContainer` and even start containers at all

Reply:
This makes it even faster 👍